src/lib/sketch/About.svelte

```diff
@@ -38,13 +38,13 @@
         </ExternalLink>
       </li>
       <li>
-        With great thanks to ATIP's various users for feedback, testing, and
+        With great thanks to our various users for feedback, testing, and
         ideas
       </li>
     </ul>
 
     <p>
-      ATIP builds on
+      Plan Your Active Travel Schemes builds on
       <ExternalLink href="https://www.openstreetmap.org/about">
         OpenStreetMap
       </ExternalLink>
```

Comment:
You can remove `osm2streets` below; it's no longer a dependency. The rest are still valid